### PR TITLE
generate talent gem

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**
* Initial gapic generation for talent
* Generate helper methods and tests
* Add kokoro configs for talent

### CODE REVIEW

File: google-cloud-logging/.autotest [hunk 1]
```diff
@@ -1,15 +0,0 @@
-# -*- ruby -*-
```
Comment (blowmage) [inline google-cloud-logging/.autotest]: Why was this file deleted?

Comment (TheRoyalTnetennba) [reply google-cloud-logging/.autotest]: It isn't included in the other gapic-only gems, and it seems less useful for gapic-only gems. What are your thoughts?

Comment (blowmage) [reply google-cloud-logging/.autotest]: I don't understand why this PR adding a new gem is removing files from an existing gem.

Comment (TheRoyalTnetennba) [reply google-cloud-logging/.autotest]: Lol didn't notice the path. Adding it back.

File: Rakefile [hunk 2]
```diff
@@ -699,15 +699,6 @@ def generate_kokoro_configs
           config = ERB.new(File.read("./.kokoro/templates/#{os_version}.cfg.erb"))
           f.write(config.result(binding))
         end
-
```
Comment (blowmage) [inline Rakefile]: Why was this removed?

Comment (TheRoyalTnetennba) [reply Rakefile]: It should've been removed earlier. For kokoro tasks there is a github side and a google3 side. In my ideal world you could be able to generate configs for both from this repo. I'm not at all sure that that is allowed. The code I deleted is an incomplete, leftover fragment from that exploration.

Comment (blowmage) [reply Rakefile]: kk
